framework/src/main/java/org/checkerframework/framework/type/poly/QualifierPolymorphism.java

```diff
@@ -73,24 +75,32 @@ static boolean isPolymorphicQualified(AnnotationMirror qual) {
     /**
      * Resolves polymorphism annotations for the given type.
      *
+     * <p>On entry, {@code type} is the type of a method declaration.
```

Comment:
These comments about declaration vs. use types aren't correct.
On entry, these are already fully substituted method signatures in a particular calling context. They are not from a method declaration. These parts should be undone.

Reply:
I'm not sure exactly what "these parts" refers to in your comment.  Can you please make the edits?